### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by pashura**

### CODE REVIEW

File: copy_cat/validators/data_type_validator.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                self.errors.append({
+                    "fieldName": test_data_object['name'],
+                    "fieldPath": design_object['location'],
+                    "xpath": test_data_object["location"],
+                    "error": error_message,
+                })
+
+        # Integer validator
+        if expected_data_type == 'Integer':
+            decimal_data_type_ = test_data_object.get('type').strip().replace("'", "")
```
Comment (akonshin) [inline copy_cat/validators/data_type_validator.py:47]: decimal_data_type_ <- copy/paste?

File: copy_cat/constants.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+IMPOSSIBLE_HUGE_LENGTH = "9999999999999999"
```
Comment (akonshin) [inline copy_cat/constants.py:2]: I like the name of this constant

File: copy_cat/validators/requirements_validator.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+class RequirementsValidator:
+    def __init__(self):
+        self.errors = []
+
+    def validate(self, test_data, schema):
+        self._validate_requirements(test_data, schema)
+        return self.errors
+
+    def _validate_requirements(self, test_data, schema):
+        for child in schema['children']:
```
[the remark this answers is not in the document]
Comment (pashura) [reply copy_cat/validators/requirements_validator.py:15]: I wouldn't use `get` anywhere it is not needed. 
`children` is always in schema object, and my option works faster ;)

File: copy_cat/constants.py (hunk 2 above)
Comment (pashura) [reply copy_cat/constants.py:2]: actually there is a mistake. I suppose it to be `IMPOSSIBLY`

**Review by akonshin — APPROVED** [review]

**Pull request merged**